src/ArcherContainer.js

```diff
@@ -331,6 +331,14 @@ export class ArcherContainer extends React.Component<Props, State> {
   render() {
     const SvgArrows = this._computeArrows();
 
+    let children;
+    console.log('Container children: ', this.props.children);
+    if (typeof this.props.children === 'function') {
+      children = this.props.children(ArcherContainerContext);
```

Comment:
Sounds good!

We had the discussion in the issue where I used the ref (for multiple containers support).
Turns we don't support it right now, is that correct? Since we create the context at the root level.